ReleaseHistory.md

```diff
@@ -21,8 +21,9 @@
 ## v4.5.2 Unreleased
 - DEP: Update SARIF SDK submodule from [dd8b7b8 to a7029a3](https://github.com/microsoft/sarif-sdk/compare/dd8b7b8..a7029a3). [Full SARIF SDK release history](https://github.com/microsoft/sarif-sdk/blob/a7029a3/ReleaseHistory.md). SDK v4.2.0.
 - DEP: Update `SEC101/003.GoogleApiKey` in `Security` validating Google API key using HTTP requests.
+- DEP: Update `SEC101/006.GitHubLegacyPat` in `Security` validating GitHub PAT using HTTP requests.
```

Comment:
>DEP [](http://example.com/codeflow?start=2&length=3)

You need to explicitly note that we've removed 'Oktokit.dll'. The reason is that some consumers might explicitly reference this file name in their utilization of our package.

Reply:
updated.